config/config.go

```diff
@@ -175,10 +175,15 @@ type RepositoryConfigFile struct {
 }
 
 type Repository struct {
-	Name        string
-	Overlays    []string `yaml:",flow"`
-	Development string
-	Master      string
+	Name            string
+	Overlays        []string `yaml:",flow"`
```

Comment:
Why is there no name in the `yaml:` here?

Reply:
I guess it's smart enough to get by without that and picks the field name.